deployments/helm-chart-dos-arbitrator/templates/controller-deployment.yaml

```diff
@@ -14,6 +14,9 @@ spec:
       labels:
         app: {{ include "arbitrator.name" . }}
     spec:
+      securityContext:
```

Comment:
I think we should probably take care of the App Protect Arbitrator separately from this PR

Reply:
Removed AppProtect from this PR. Should I open a separate PR myself for that change?

Reply:
I think we're gonna move some stuff around then updated the AP chart, thanks!